src/Text/Pandoc/Readers/RST.hs

```diff
@@ -770,6 +771,26 @@ tableDirective top _fields body = do
                                   aligns' widths' header' rows'
        _ -> return mempty
 
+listTableDirective :: PandocMonad m => String -> [(String, String)] -> String -> RSTParser m Blocks
+listTableDirective top _fields body = do
+  bs <- parseFromString parseBlocks body
+  title <- do { parseFromString (trimInlines . mconcat <$> many inline) top }
+  let rows = takeRows $ B.toList bs
+      numOfCols = length $ rows !! 0
+      headerRows = case lookup "header-rows" _fields of
+                     Just x -> read x :: Int
+                     Nothing -> 0
+  return $ B.singleton $ Table (B.toList title)
+                           (replicate numOfCols AlignDefault)
+                           (replicate numOfCols 0)
+                           (if headerRows > 0 then rows !! 0 else replicate numOfCols [])
```

Comment:
I'm not sure if this is the right place for this question, but do you know when this commit will be merged?  I'm right in the middle of producing critical docs, and the lack of list-table for rst is proving to be a big hindrance.